docs/openapi/schemas.yaml

```diff
@@ -1290,14 +1290,19 @@ ImportJobRequest:
     importScript:
       description: Optional import.js mapping file
       type: string
-      format: base64
+      format: binary
+    customHeaders:
+      description: Optional custom headers to send with each HTTP request
+      type: object
   example:
     urls:
       - 'https://example.com/en/page1'
       - 'https://example.com/fr/page2'
```

Comment:
Is this example still ok?
Maybe:
`['https://example.com/en/page1', 'https://example.com/fr/page2']`

Reply:
This is how it renders in Redocly, which I think is ok:

<img width="520" alt="Screenshot 2024-09-10 at 2 53 13 PM" src="https://github.com/user-attachments/assets/60f7f9ed-fc9b-4dd4-a5fd-ce03cd8f6814">